DOCUMENTATION.md

```diff
@@ -2754,12 +2879,13 @@ Example casting:
 SELECT * FROM someMap WHERE this = CAST(? AS INTEGER)
 ```
 
-#### An Example of Casting
+#### 8.7.9.2. An Example of Casting
 
-Since Node.js client uses double as default number type, to compare with an integer based column you need to use casting.
+Since Node.js client's number type is like a double, to compare with an integer-like column, you need to use casting.
+Alternatively, you can use `Long` objects to compare with columns with `BIGINT` type.
```

Comment:
```suggestion
Alternatively, you can use `Long` objects to compare columns with `BIGINT` type.
```
Maybe this a bit more clear, WDYT?

Reply:
I went with a different one:

Alternatively, you can use `Long` objects where you have a column with `BIGINT` type, because `Long` objects correspond to the `BIGINT` type.